modules/OCPP/OCPP.cpp

```diff
@@ -752,6 +752,42 @@ void OCPP::ready() {
             this->p_ocpp_generic->publish_boot_notification_response(everest_boot_notification_response);
         });
 
+    this->charge_point->register_session_cost_callback(
+        [this](const ocpp::RunningCost& session_cost,
+               const uint32_t number_of_decimals) -> ocpp::v16::DataTransferResponse {
+            const types::session_cost::SessionCost cost =
+                conversions::create_session_cost(session_cost, number_of_decimals, {});
+            ocpp::v16::DataTransferResponse response;
+            if (this->p_session_cost == nullptr) {
```

Comment:
nit: `p_session_cost` cannot really be null, since providing an interface implementation is not optional for a module.

Reply:
ok i'll remove it. on your risk ;)